packages/config/README.md

```diff
@@ -101,6 +109,13 @@ const libConf = config.librarySync('bem-components');
 console.log(libConf); // library config
 ```
 
+### levels
+```js
+const config = require('@bem/sdk.config')();
+const desktopSet = config.levels('desktop');
```

Comment:
may be `levelsSync`?

Reply:
I'd add a note about sync nature of this method